sensing/pointcloud_preprocessor/include/pointcloud_preprocessor/filter.hpp

```diff
@@ -133,7 +133,6 @@ class Filter : public rclcpp::Node
     const std::string & filter_name = "pointcloud_preprocessor_filter",
     const rclcpp::NodeOptions & options = rclcpp::NodeOptions());
 
-protected:
   struct TransformInfo
   {
```

Comment:
Can you define in another header file?
Since voxel grid filter is a basic function, we would like to make it a library so that it can be called from other packages in the future.

Reply:
Addressed in https://github.com/autowarefoundation/autoware.universe/pull/3679/commits/6b859ec9534f4bea93d5c58c4781be8be306fe98.